its/fixtures/pipeline-sonarcloud-v1-cli-6.2.1.4610-unix.yml

```diff
@@ -2,23 +2,24 @@
 # Config: {"os":"unix","version":{"extension":"sonarcloud","version":1},"scanner":{"type":"cli","version":"6.2.1.4610"}}
 
 trigger: none
+pr: none
 pool:
   vmImage: ubuntu-latest
 variables:
   system.debug: true
 steps:
-  - task: SonarcloudPrepare@1
+  - task: SonarCloudPrepareTest@1
```

Comment:
in the end you changed the task names from `SonarCloudPrepare` to `SonarCloudPrepareTest`.  The reasoning is to allow for test and dogfood tasks to co-exist for the same org. Is this captured somewhere?

Reply:
I've just added a comment where we store the suffix value (`its/constant.ts`)